content/devops-israel/drone-lambda/index.md

```diff
@@ -0,0 +1,99 @@
+---
+author: "devops-israel"
+date: 2017-10-22T08:00:35+01:00
+image: "omerxx/drone-lambda-plugin"
+repo: "omerxx/drone-lambda-plugin"
+tags: [ deploy, aws, lambda ]
+logo: amazon_lambda.svg
+title: AWS Lambda Deploy
+---
+
+The plugin utilizes AWS go-sdk to update an existing function's code; build your code, zip it with dependencies and upload it to S3. Then trigger the plugin for deploy.
+
+## Usage
```

Comment:
Can you remove these headings so the formatting of this will look similar to other docs?

Reply:
Sure, Done.